.ci/install.sh

```diff
@@ -18,3 +18,5 @@ elif [[ "${BUILD_MINGW}" == ON ]]; then
 fi
 
 pip install --user --upgrade cpp-coveralls neovim
+pip-3.3 install --user --upgrade neovim
```

Comment:
I thought just adding `python3-pip` to the installed APT packages should be enough, but looks like Ubuntu 12.04 doesn't have this package, for whatever reason. The following should work:

 * Add `python3-setuptools`, `python3-dev` to APT packages (depends on https://github.com/travis-ci/apt-package-whitelist/issues/771)
 * Run `easy_install3 --user neovim`